README.md

```diff
@@ -43,19 +43,18 @@ they wish to maintain it going forward.
 See
 [COMMAND_REFERENCE.md](https://github.com/GoogleCloudPlatform/snapshot-debugger/blob/main/snapshot_dbg_cli/COMMAND_REFERENCE.md).
 
-## Download the Snapshot Debugger CLI
+## Installing the Snapshot Debugger CLI
 
-Clone the debugger to your local environment, or to your Cloud Shell
+Install the debugger cli in your local environment, or in your Cloud Shell
 $HOME directory. See [Using Cloud
 Shell](https://cloud.google.com/shell/docs/using-cloud-shell) for information on
 using Cloud Shell.
 
 Note: When using the Snapshot Debugger in Cloud Shell you will be asked to
 Authorize using your account credentials.
 
-1. Clone the Snapshot Debugger CLI
 ```
-git clone https://github.com/GoogleCloudPlatform/snapshot-debugger.git
+python3 -m pip install snapshot-dbg-cli
 ```
 
```

Comment:
It might be worth adding a note about how to run the cli:

snapshot-dbg-cli script will be installed

python3 -m snapshot_dbg_cli is still an option

Reply:
Thanks, I've updated the wording, PTAL.